contrib/cirrus/runner.sh

```diff
@@ -134,28 +229,19 @@ dotest() {
         # does not return
     fi
 
-    # 'logformatter' script makes test logs readable; only works for some tests
-    case "$testsuite" in
-        integration|system)  output_filter=logformatter ;;
-        *)                   output_filter="cat"        ;;
-    esac
```

Comment:
Unnecessary: `dotest()` is only ever invoked for system or integration tests, nothing else.

Reply:
Correct, at the time I wasn't aware the logformatter worked with system-tests.

Reply:
(I was referring to my own mistake in #7924 -- I added the above `case` statement in `dotest()`, not realizing that the `*` condition would never be triggered).